Splunk_TA_paloalto/default/tags.conf

```diff
@@ -20,27 +21,24 @@ start = enabled
 
 [eventtype=pan_traffic_end]
 network = enabled
-session = enabled
-end = enabled
+communicate = enabled
```

Comment:
1. Based on our discussions, it sounds like eventtype `pan_traffic_start` should not be tagged as the Session datamodel. Should we remove the tags for that eventtype?
2. Do we need to tag `pan_traffic` and `pan_traffic_end`? Seems like just `pan_traffic_end` would be enough?

Reply:
Yes, pan_traffic_end would be enough. And if you are removing tags from pan_traffic_start then we suggest keeping tags for pan_traffic as they will be used to map tags to 'start' events as well.

Reply:
I see, in that case I can add Network Traffic tags to pan_traffic_start so those events are also tagged. I thought from the PR that the intention was to not tag pan_traffic_start, but based on this comment it sounds like we should.

Let me know if I should not add Network Traffic tags for pan_traffic_start.